src/dcd_mapping/vrs_map.py

```diff
@@ -391,14 +397,21 @@ def _map_genomic(
         sequence_id,
         True,
     )
-    post_mapped_genomic = _construct_vrs_allele(
-        post_mapped_hgvs_strings,
-        AnnotationLayer.GENOMIC,
-        None,
-        False,
-    )
 
-    if pre_mapped_genomic and post_mapped_genomic:
+    # genomic post-mapped variants are skipped if they don't fall within aligned region of target sequence
+    # so we can have a pre-mapped genomic variant without a post-mapped genomic variant
+    if post_mapped_hgvs_strings:
+        post_mapped_genomic = _construct_vrs_allele(
+            post_mapped_hgvs_strings,
+            AnnotationLayer.GENOMIC,
+            None,
+            False,
+        )
+    else:
+        # TODO add error to MappedScore
+        post_mapped_genomic = None
+
+    if pre_mapped_genomic:
```

Comment:
Similar to a few comments above. We know whether `pre_mapped_genomic` is defined on line 394, so we should be able to check if it is None and return immediately. Here I'm less inclined to leave as is for readability reasons because of the addition of the if/else block above this one.

So our flow could look something like:
```
pre_mapped_genomic = _construct_vrs_allele

if pre_mapped_genomic is None:
      return None

try to do post mapped genomic
return the mapped score
```